l10n_it_fatturapa/bindings/fatturapa_v_1_2.py

```diff
@@ -1945,7 +1945,9 @@ class DataFatturaType (pyxb.binding.datatypes.date):
 DataFatturaType._CF_minInclusive = pyxb.binding.facets.CF_minInclusive(
     value_datatype=DataFatturaType,
     value=pyxb.binding.datatypes.date('1970-01-01'))
-DataFatturaType._InitializeFacetMap(DataFatturaType._CF_minInclusive)
+# FIXME: if a date in the xml has the timezone, pyxb is unable to check this rule,
+# so is disabled for now
+#DataFatturaType._InitializeFacetMap(DataFatturaType._CF_minInclusive)
```

Comment:
Come detto anche in https://github.com/OCA/l10n-italy/pull/904#pullrequestreview-196986049, citerei questo problema nella ROADMAP

Reply:
Come dicevi anche tu in https://github.com/OCA/l10n-italy/pull/904#issuecomment-458095069 ci sarebbe da controllare se questo possa modificare il comportamento degli XML che noi generiamo.
Ne ho generata una nel runbot di questa PR e il formato è corretto, però forse serve qualche modifica perché al momento ci affidiamo al formato con cui la data è salvata su DB (che per com'è adesso, va bene): 

- prima di questa PR il controllo del formato era demandato a PyXB
- ora mi sa che dovremmo farlo noi in https://github.com/OCA/l10n-italy/blob/8.0/l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py#L489

Reply:
> ora mi sa che dovremmo farlo noi in https://github.com/OCA/l10n-italy/blob/8.0/l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py#L489

Non vedo che problema ci possa essere

Reply:
> > ora mi sa che dovremmo farlo noi in https://github.com/OCA/l10n-italy/blob/8.0/l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py#L489
> 
> Non vedo che problema ci possa essere

Se `date_invoice` non è nel formato corretto, verrà rifiutata da SDI.
Prima il controllo era fatto da PyXB (e a quanto pare era troppo restrittivo), ora non più